### PR TITLE
fixes #48

### LINKED ISSUE #48: Make fakeIncludes.html

### PR DESCRIPTION
**Pull request by marirumo**

### CODE REVIEW

File: functions/fakeIncludes.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                </div>
+            </section>
+
+            <section id="examples">
+                <h2 class="section-title">Ejemplos</h2>
+
+                <article>
+                    <pre>
+              <code class="code js">
+                let numbers = [1, 2, 3]
```
Comment (eebalcarcel) [inline functions/fakeIncludes.html]: Remove this or pass it as an argument to the function

File: functions/fakeIncludes.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                    corresponda
+                </p>
+            </section>
+
+            <section id="syntax">
+                <h2 class="section-title">Sintaxis</h2>
+                <div>
+                    <pre>
+            <code class="code syntax js">
+                function fakeIncludes(array, elem, fromIndex)
```
Comment (eebalcarcel) [inline functions/fakeIncludes.html]: Remove the function keyword and add the square brackets to fromIndex (it's an optional parameter). It should be like this fakeIncludes(array, elem[, fromIndex])

File: functions/fakeIncludes.html (hunk 1 above)
Comment (marirumo) [reply functions/fakeIncludes.html]: Done

File: functions/fakeIncludes.html (hunk 2 above)
Comment (marirumo) [reply functions/fakeIncludes.html]: Done!

**Pull request merged**